packages/validator/src/commands/convertCmd.ts

```diff
@@ -9,27 +10,52 @@ import { invalidConversionMessage } from '../utils/messages';
 const oas2ois = 'OAS2OIS.json';
 const ois2config = 'OIS2Config.json';
 
-if (process.env.npm_config_template) {
-  console.log(
-    JSON.stringify(convert(process.env.npm_config_specs || process.argv[3], process.env.npm_config_template), null, 2)
-  );
+const args = yargs(hideBin(process.argv))
+  .option('from', {
+    description: 'Name of the source airnode specification format',
+    demandOption: 'to',
+    default: '',
+    string: true,
+  })
+  .option('to', {
+    description: 'Name of the target airnode specification format',
+    demandOption: 'from',
+    default: '',
+    string: true,
+  })
+  .option('template', {
+    description: 'Path to validator template file',
+    alias: 't',
+    default: '',
+    string: true,
+  })
```

Comment:
If I understand it correctly you can choose to either provide a `template` or `from` and `to`, right? In that case you can use [`conflicts`](https://github.com/yargs/yargs/blob/master/docs/api.md#conflictsx-y) to enforce that.

Reply:
Since `from` and `to` have default values set, using `conflicts` would cause to print error every time the command is ran with `template` argument